cloudformation/cid-crdc-resources.yaml

```diff
@@ -0,0 +1,709 @@
+      -
+        Label:
+          default: "AWS Config Logging"
+        Parameters:
+          - PConfigLoggingBucket
+          - PConfigLoggingAccountID 
+      -
+        Label:
+          default: "Athena Configuration (DO NOT CHANGE)"
+        Parameters:
+          - PAthenaWorkgroup
+          - PAthenaDatabase
+          - PAthenaTable
+          - PAthenaQueryResultBucketPrefix
+      
+    ParameterLabels:
+      PAthenaWorkgroup:
+        default: "Athena workgroup"
+      PAthenaDatabase:
+        default: "Athena database"
+      PAthenaQueryResultBucketPrefix:
+        default: "Prefix of the Athena query results bucket"
+      PAthenaTable:
+        default: "Athena table for AWS Config data"
+      PConfigLoggingBucket:
+        default: "AWS Config bucket"
+      PConfigLoggingAccountID:
+        default: "Config bucket account"
+      PQuickSightUser:
+        default: "User name of QuickSight user (as displayed in QuickSight admin panel)"
+      
+Parameters:
+  PQuickSightUser:
+    Type: String
+    MinLength: 1
+    Default: "Replace with QuickSight user"
+    Description: See https://quicksight.aws.amazon.com/sn/admin#users
+
+  PConfigLoggingBucket:
+    Type: "String"
+    Default: "your-config-bucket"
+    Description: "Name of the Amazon S3 bucket that collects AWS Config data (Required)"
+    AllowedPattern: '^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$'
+    ConstraintDescription: "Required"
+  
+  PConfigLoggingAccountID:
+    Type: "String"
+    Default: "123412341234"
+    Description: "AWS Account ID that contains the bucket holding AWS Config snapshots (Required)"
+    MaxLength: 12
+    MinLength: 12
+    AllowedPattern: '^\d{12}$'
+    ConstraintDescription: "Required"
+
+  PAthenaDatabase:
+    Type: "String"
+    # The name cannot have the symbol '-' and must have lowercase character. '_' seems accepted
+    Default: "cid_crcd_database"
+    Description: "The Athena/Glue database for the CID-CRCD dashboard (Required)"
+    MinLength: 1
+    ConstraintDescription: "Required"
+
+  PAthenaQueryResultBucketPrefix:
+    Type: "String"
+    Default: "cid-crcd-athena-query-results" 
+    Description: "The Athena query result bucket for the workgroup, Account ID and Region will be added to the name by this tempalte (Required)"
+    # 64 characters in the bucket name, but automatically the template will add 2 dashes, 12 digit account number and region up to 14 characters ap-southeast-4, ap-northeast-1
+    # that leaves 36 character for the prefix
+    AllowedPattern: '^[a-z0-9][a-z0-9-]{1,33}[a-z0-9]$'
+    ConstraintDescription: "Required"
+
+  PAthenaWorkgroup:
+    Type: "String"
+    Default: "cid-crcd-dashboard"
+    Description: "The Athena workgroup for the CID-CRCD dashboard (Required)"
+    MinLength: 1
+    ConstraintDescription: "Required"
+
+  PAthenaTable:
+    Type: "String"
+    Default: "cid_crcd_config"
+    Description: "The Athena table that contains the AWS Config data for the dashboard (Required)"
```

Comment:
Would be nice to explain customer how they find this table

Reply:
Adjusted Description. This is the name that will be assigned to the table, actually... it does not exist yet.

Reply:
Resolving conversation as agreed with Iakov